api/AudioBufferSourceNode.json

```diff
@@ -347,16 +347,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBufferSourceNode/onended",
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "30"
```

Comment:
Everywhere one looks closely, something is a bit broken: https://github.com/mdn/browser-compat-data/issues/6656

But anyway, the `onended` attribute was added in https://chromium.googlesource.com/chromium/src/+/976bdc021a39e160f5d0d4da9b0d477eb4232e69 for AudioBufferSourceNode and OscillatorNode. That was indeed M30 per https://www.chromium.org/developers/calendar.